contracts/Router.sol

```diff
@@ -0,0 +1,261 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.0) (finance/PaymentSplitter.sol)
```

Comment:
This doesn't seem right given that we have modified the contract, also we usually use the Apache-2.0 license. I'd suggest updating to include all contributors:

 ```suggestion
// Copyright OpenZeppelin, BigchainDB GmbH and Ocean Protocol contributors
// SPDX-License-Identifier: (Apache-2.0 AND MIT)
```

Reply:
Although, this isn't needed if we just inherit form `PaymentSplitter.sol` which would be cleaner. Then we should just change the license identifier and copyright info to our own in this file.

Reply:
It would follow the pattern that we have used previously: [example](https://github.com/oceanprotocol/contracts/blob/main/contracts/pools/balancer/BFactory.sol#L2)